patches/server/0273-Verify-player-key-early.patch

```diff
@@ -0,0 +1,96 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "FivePB (Xer)" <admin@fivepb.me>
+Date: Thu, 28 Jul 2022 11:36:36 +0000
+Subject: [PATCH] Verify player key early
+
+This patch refactors authentication with the player key in a way
+that forces the key to be verified early using the (currently unused)
+player UUID provided in the Server hello packet.
+This helps verify connections early and may reduce requests to the session
+service
```

Comment:
This patch seems like unnecessarily high-conflict diff for too little benefit in a normal server scenario, since it doesn't help in any other important way